beacon_node/network/src/sync/range_sync/batch.rs

```diff
@@ -525,3 +530,28 @@ impl<E: EthSpec> std::fmt::Debug for BatchState<E> {
         }
     }
 }
+
+impl<E: EthSpec> BatchState<E> {
+    /// Creates a unicode visualisation for the batch state to display in logs for quicker and
+    /// easier recognition
+    /// NOTE: May require nerd-fonts
+    ///
+    /// The current icons are:
+    /// - Empty/Uninitialized: 
+    /// - Downloading: ⏬
+    /// - Awaiting Download: 📥
+    /// - Awaiting Validation:⏳
+    /// - Failed: ❌
+    /// - AwaitingProcessing: 
```

Comment:
`Empty/Uninitialized` and `AwaitingProcessing` aren't rendering for me here - are they emojis? This list don't cover all variants below, (missing `Processing`, `Poisoned`)